pkg/docker/config/config_linux.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/containers/image/v5/internal/pkg/keyctl"
+	"github.com/containers/image/v5/internal/pkg/keyctl" //nolint
```

Comment:
Now that the subpackage is internal anyway, and this is the only user, maybe we can just remove the “Deprecated:” comment instead.

Reply:
Will do, thanks!